.props/_Nupkg.props

```diff
@@ -108,4 +100,11 @@
     </Task>
   </UsingTask>
 
+  <PropertyGroup>
+    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
+  </PropertyGroup>
+  <ItemGroup>
+    <EmbeddedFiles Include="$(GeneratedAssemblyInfoFile)"/>
+  </ItemGroup>
+
```

Comment:
what does this do?

Reply:
This was one of the suggestions to embed the attributes files. in theory, it would solve the deterministic thing. but that didnt work. i will remove it

Reply:
just reverted!